zebra-rpc/src/methods.rs

```diff
@@ -188,4 +191,4 @@ pub struct GetBlockChainInfo {
 /// Response to a `sendrawtransaction` RPC request.
 ///
 /// A JSON string with the transaction hash in hexadecimal.
-pub struct SentTransactionHash(String);
+pub struct SentTransactionHash(#[serde(with = "hex")] transaction::Hash);
```

Comment:
I managed to shorten the attribute and also make it future-proof if we end up having to `derive(Deserialize)` for it.